### PR TITLE
Update many-rs to latest

### PR DESCRIPTION
**Pull request by hansl**
Includes refactors to identity and async client.

### CODE REVIEW

File: src/ledger/src/multisig.rs [hunk 1]
```diff
@@ -109,42 +109,40 @@ fn submit_send(
         timeout,
         execute_automatically,
     } = multisig_arg;
-    let symbol = crate::resolve_symbol(&client, symbol)?;
-
-    if client.id.identity.is_anonymous() {
-        Err(ManyError::invalid_identity())
-    } else {
```
Comment (fmorency) [inline src/ledger/src/multisig.rs:116]: That reminds me that we should add `#[deny_anonymous]` to additional methods in `many-rs`.

https://github.com/liftedinit/many-rs/issues/197

**Review by fmorency — APPROVED** [review]
Lgtm, thanks!

**Review by Denommus — APPROVED** [review]
LGTM

**Comment by Denommus** [comment]
I'm merging this because I need this on main to proceed with my work and @hansl is out this week.

**Comment by Denommus** [comment]
I'm merging this because I need this on main to proceed with my work and @hansl is out this week.

**Pull request merged**